frontend-react/src/components/MessageTracker/MessageDetails.test.tsx

```diff
@@ -167,27 +168,76 @@ jest.mock("react-router-dom", () => ({
 describe("RSMessageDetail component", () => {
     test("url param (messageId) feeds into network hook", () => {
         mockUseMessageDetails.mockReturnValueOnce({
-            messageDetails: MOCK_EMPTY_MESSAGE_DETAIL as RSMessageDetail,
+            messageDetails: MOCK_EMPTY_MESSAGE_DETAIL,
         });
         render(<MessageDetails />);
         expect(mockUseMessageDetails).toHaveBeenCalledWith(TEST_ID);
     });
 
-    test("renders expected content", async () => {
+    test("renders expected content", () => {
         mockUseMessageDetails.mockReturnValueOnce({
-            messageDetails: MOCK_MESSAGE_DETAIL as RSMessageDetail,
+            messageDetails: DEFAULT_MESSAGE_DETAIL,
         });
         render(<MessageDetails />);
 
-        expect(screen.getByText(/Message ID/)).toBeInTheDocument();
-        expect(screen.getByText(/12-234567/)).toBeInTheDocument();
-        expect(screen.getByText(/somebody 1/)).toBeInTheDocument();
+        expect(screen.getByText("Message ID")).toBeVisible();
+        expect(screen.getByText(/12-234567/)).toBeVisible();
+        expect(screen.getByText(/Submitter/)).toBeVisible();
+        expect(screen.getByText(/somebody 1/)).toBeVisible();
+        expect(screen.getByText(/Incoming Report ID/)).toBeVisible();
         expect(
             screen.getByText(/29038fca-e521-4af8-82ac-6b9fafd0fd58/)
-        ).toBeInTheDocument();
+        ).toBeVisible();
+        expect(screen.getAllByText("Date/Time Submitted")[0]).toBeVisible();
+        expect(screen.getByText("9/28/2022, 12:00:00 am")).toBeVisible();
+        expect(screen.getByText("File Location")).toBeVisible();
+        expect(screen.getByText("RECEIVE")).toBeVisible();
+        expect(screen.getByText("/ simple_report.csvuploader")).toBeVisible();
```

Comment:
Just so we don't have to check explicitly for a slash and space (which feels a little fragile), we could just render the slash with horizontal:

```jsx
<span className="font-mono-sm border-1px bg-primary-lighter radius-md padding-top-4px padding-bottom-4px padding-left-1 padding-right-1">{folderLocation}</span>
<span className="padding-x-1">/</span>
<span>{sendingOrg}</span>
```

Reply:
I will update the slash after testing different scenarios.